include/ck/tensor_operation/gpu/device/device_reduce.hpp

```diff
@@ -16,9 +16,11 @@ namespace device {
 template <typename InElementwiseOperation, typename AccElementwiseOperation>
 struct DeviceReduce : public BaseOperator
 {
-    virtual size_t GetWorkspaceSizeInBytes(const std::vector<int>& inLengths)
+    virtual size_t GetWorkspaceSizeInBytes(const std::vector<int> inLengths,
```

Comment:
std::size_t

Reply:
Agree!    I also insist for an obvious API to not use reference for parameter (`std::vector<int>& inLengths`),  direct value passing should be used (`std::vector<int> inLengths`),  even though it is for container type

Reply:
done